modules/accessing-an-example-cluster-node-tuning-operator-specification.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-tuning-operator.adoc
+
+[id='accessing-an-example-node-tuning-operator-specification-{context}']
+= Accessing an example node tuning Operator specification
```

Comment:
Why the odd capitalization? _node tuning Operator_ looks odd.

Reply:
Perhaps this is an attempt to distinguish the node-tuning Operator itself vs. the logical operators?  I don't mind having the node-tuning operator lowercase, but it needs to be consistent across the OCP documentation with other sections that talk about [Oo]perators.

Reply:
At some point, we decided to always capitalize [Operator](https://github.com/openshift/openshift-docs/blob/enterprise-4.0/contributing_to_docs/term_glossary.adoc#operator).

Reply:
And also the names of Operators. Node Tuning Operator is one of the examples listed here:

https://github.com/openshift/openshift-docs/blob/enterprise-4.0/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
@adellape Yeah, I need to switch to its full proper name